ddtrace/mocktracer/mockspan.go

```diff
@@ -201,6 +201,9 @@ func (s *mockspan) Finish(opts ...ddtrace.FinishOption) {
 	if cfg.Error != nil {
 		s.SetTag(ext.Error, cfg.Error)
 	}
+	if cfg.NoDebugStack {
+		s.SetTag(ext.ErrorStack, "<mock no debug stack>")
```

Comment:
Maybe this would be better
```suggestion
		s.SetTag(ext.ErrorStack, "<debug stack disabled>")
```
Otherwise it's hard to tell the difference between no debug stack (empty string) and `"<mock no debug stack>"`